src/operator/nn/mkldnn/mkldnn_fully_connected.cc

```diff
@@ -26,6 +26,7 @@
 
 #if MXNET_USE_MKLDNN == 1
 #include "mkldnn_fully_connected-inl.h"
+#include <unordered_map>
```

Comment:
where is this needed?

Reply:
Why don't you make the effort to open this very file. You assume that I make changes for no reason. This kind of review is not useful.

Reply:
@larroy no need to take it personal mate. It's alright to ask questions.
@apeforest 

https://github.com/apache/incubator-mxnet/blob/80309ff1dfd0a7ea052956351d2f91ab597bacf7/src/operator/nn/mkldnn/mkldnn_fully_connected.cc#L184

Reply:
@larroy Please don't be so defensive. I simply didn't understand why if a missing header file in the code would cause warning rather than just failing to compile. It would be great if you could answer my question and thanks @ChaiBapchya for pointing me to the location. But again, why it would pass compilation earlier if that needed STL header was missing?